demos/common/cpp/models/include/models/detection_model_yolo.h

```diff
@@ -53,8 +61,11 @@ class ModelYolo : public DetectionModel {
     /// during postprocessing (only one of them should stay). The default value is 0.5
     /// @param labels - array of labels for every class. If this array is empty or contains less elements
     /// than actual classes number, default "Label #N" will be shown for missing items.
+    /// @param rawAnchors - vector of anchors coordinates. Required for YOLOv4, for other versions it may be omitted.
+    /// @param rawMasks - vector of masks values. Required for YOLOv4, for other versions it may be omitted.
     ModelYolo(const std::string& modelFileName, float confidenceThreshold, bool useAutoResize,
-        bool useAdvancedPostprocessing = true, float boxIOUThreshold = 0.5, const std::vector<std::string>& labels = std::vector<std::string>());
+        bool useAdvancedPostprocessing = true, float boxIOUThreshold = 0.5, const std::vector<std::string>& labels = std::vector<std::string>(),
+        const std::vector<float>& rawAnchors = std::vector<float>(), const std::vector<int64_t>& rawMasks = std::vector<int64_t>());
```

Comment:
Remove the `raw` suffix from parameter names - it has no sense

Reply:
Done